### ISSUE TITLE
Support for HistGradientBoostingRegressor

### ISSUE THREAD

**Issue by ogencoglu** [uxlfoundation/scikit-learn-intelex#2372]
Histogram-based Gradient boosting support would be great:

[HistGradientBoostingRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingRegressor.html#sklearn.ensemble.HistGradientBoostingRegressor)
